subprojects/core/src/integTest/groovy/org/gradle/configuration/ExecuteUserLifecycleListenerBuildOperationIntegrationTest.groovy

```diff
@@ -401,18 +399,15 @@ class ExecuteUserLifecycleListenerBuildOperationIntegrationTest extends Abstract
 
         buildFile << addGradleListeners('root project script')
         buildFile << addProjectListeners('root project script', ':')
-        buildFile << addProjectListeners('root project script', ':sub')
         applyInlinePlugin(buildFile, 'Project', addProjectListeners('root project plugin'))
         applyScript(buildFile, scriptFile)
 
         subBuildFile << addGradleListeners('sub project script')
-        subBuildFile << addProjectListeners('sub project script', ':')
```

Comment:
The listener is registered in a subproject for the root. When the configuration phase reaches this point, the root project is already evaluated, leading to the freshly introduced build failure.